nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/FetchAzureDataLakeStorage.java

```diff
@@ -78,12 +79,18 @@ public class FetchAzureDataLakeStorage extends AbstractAzureDataLakeStorageProce
             .defaultValue("0")
             .build();
 
+    private static final List<PropertyDescriptor> properties = Collections.unmodifiableList(Arrays.asList(
+        FILESYSTEM_WITH_DEFAULT,
+        DIRECTORY_WITH_DEFAULT,
+        FILE,
+        ADLS_CREDENTIALS_SERVICE,
+        RANGE_START,
+        RANGE_LENGTH,
+        NUM_RETRIES
```

Comment:
Similar to `FetchAzureBlobStorage_v12`: `FILE` and `RANGE_*`properties should not be split, I believe.